src/renderer/utils/InstallManager.tsx

```diff
@@ -256,7 +258,64 @@ export class InstallManager {
             }
         }
 
-        const destDir = Directories.inCommunity(addon.targetDirectory);
+        // Find incompatible add-ons
+        // This iterates through the first level of folders of the  MSFS Community folder looking for the manifest.json
+        // file. It compares the manifest.json file content with the configured incompatible add-ons (data.ts) and if it
+        // finds one or more matches, it will issue a warning.
+        console.log('Searching incompatible add-ons');
+        const manifestFileName = 'manifest.json';
+        const comDir = Directories.communityLocation();
+        const incompatibleAddons: AddonIncompatibleAddon[] = [];
+        try {
+            const addonFolders = fs.readdirSync(comDir);
+            for (const entry of addonFolders) {
+                const filePath = path.join(comDir, entry);
+                const stat = fs.statSync(filePath);
+                if (stat.isDirectory()) {
+                    const dirEntries = fs.readdirSync(filePath);
+                    if (dirEntries.includes(manifestFileName)) {
+                        const manifest = JSON.parse(fs.readFileSync(path.join(filePath, manifestFileName), 'utf8'));
+                        for (const item of addon.incompatibleAddons) {
+                            if ((!item.title || manifest.title === item.title) &&
```

Comment:
**style:** this if statement is very hard to read. I would split it into multiple conditions

**issue:** the version checking will fail any time the author updates their package, and will not allow detecting a range or set of versions. a package like `semver` should be used instead

Reply:
will change the style for better readability

An improved version check I wanted to do later in a separate PR. I like to iterate improvements after something was in production and we gathered some feedback and learnings.